connectivity/tests/host.go

```diff
@@ -42,10 +42,25 @@ func (s *podToHost) Run(ctx context.Context, t *check.Test) {
 						continue
 					}
 
+					switch {
+					case addr.Type == "InternalIP":
+						{
+							addrType = "internal-ip"
+						}
+					case addr.Type == "ExternalIP":
+						{
+							addrType = "external-ip"
+						}
+					case addr.Type == "Hostname":
+						{
+							addrType = "hostname"
+						}
+					}
```

Comment:
How about using the `NodeAddressType` constants defined in `k8s.io/api/core/v1`?

additional nit: no need for the curly braces in Go:

```suggestion
					switch {
					case addr.Type == corev1.NodeInternal:
						addrType = "internal-ip"
					case addr.Type == corev1.NodeExternalIP:
						addrType = "external-ip"
					case addr.Type == corev1.NodeHostName:
						addrType = "hostname"
					}
```

Reply:
Good proposal. Thank you.

Reply:
Sorry, I forgot to mention you'll also need to add `corev1 k8s.io/core/v1` to the `import` section at the top of the file. Please also squash the second commit into to first one.